Homework/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Tyrone Biggums</title>
+		<link rel="stylesheet" href="css/styles.css">
+		<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css"/>
+		<link rel="stylesheet" href="http://colorzilla.com/gradient-editor/#b4ddb4+0,83c783+17,52b152+33,008a00+67,005700">
+
+	</head>
+	<body>
+
+		<nav>
+			<a href="index.html">About Me |</a>
+			<a href="portfolio.html">Portfolio</a>
+			
+		</nav>
+
+		<h1 class="text-color">SHAZAM!!!</h1>
+		<hr>
+		<h3 class="text-color">There's something you should know about me, Joe Rogan...
+			<i class="fa fa-hand-spock-o"></i>
+		</h3>
+
+		<p class="text-color">My name is Tyrone Biggums. I speak at elementary schools regarding the perils of using drugs. 
+ 		In my free time, I enjoy:
+ 			<ul class="text-color">
+ 				<li>Stealing radios</li>
+ 				<li>Dancing</li>
+ 				<li>Peanut butter and crack sandwiches</li>
+ 			</ul> 
+			
+		</p>
+
+		<div class="center">
+			<p>
```

Comment:
:hammer: These `<p>` tags are unnecessary here. I would remove them, then see what things look like. On the page, those `<p>` tags aren't actually behaving the way I think you want them too.